src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableSortedDictionary.cs

```diff
@@ -164,14 +164,14 @@ public static class ImmutableSortedDictionary
         /// <summary>
         /// Returns an immutable copy of the current contents of the builder's collection.
         /// </summary>
-        /// <param name="builder">The builder to create the immutable map from.</param>
+        /// <param name="source">The builder to create the immutable map from.</param>
         /// <returns>An immutable map.</returns>
         [Pure]
-        public static ImmutableSortedDictionary<TKey, TValue> ToImmutableSortedDictionary<TKey, TValue>(this ImmutableSortedDictionary<TKey, TValue>.Builder builder)
+        public static ImmutableSortedDictionary<TKey, TValue> ToImmutableSortedDictionary<TKey, TValue>(this ImmutableSortedDictionary<TKey, TValue>.Builder source)
```

Comment:
What about other APIs from https://github.com/dotnet/corefx/issues/21055?

For example: https://github.com/dotnet/corefx/blob/9f404b47df54bb6cd9a01b233f56f5f005241195/src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableHashSet.cs#L165
https://github.com/dotnet/corefx/blob/9f404b47df54bb6cd9a01b233f56f5f005241195/src/System.Collections.Immutable/ref/System.Collections.Immutable.cs#L403

Maybe this should be brought up and fixed separately for all the new collection APIs so we are consistent.

I am not sure if `source` is better than `builder`, but whatever option we pick, we should apply it to all the relevant places.

cc @terrajobst

Reply:
Also cc @jamesqo to verify original intent.